editor/test/integration/gui_test.clj

```diff
@@ -158,7 +158,7 @@
       (test-util/with-prop [pie :perimeter-vertices 1001]
         (is (g/error-fatal? (test-util/prop-error pie :perimeter-vertices)))))))
 
-(deftest gui-textures
+(deftest gui-textures-test
```

Comment:
Renamed some tests, as they were conflicting with function names in the same scope. Let's make sure to always suffix test names with `-test` going forward!